app/upgrades.go

```diff
@@ -227,6 +227,7 @@ func (app *StrideApp) setupUpgradeHandlers(appOpts servertypes.AppOptions) {
 			app.InterchainqueryKeeper,
 			app.RatelimitKeeper,
 			app.StakeibcKeeper,
+			app.BankKeeper,
```

Comment:
the absolute pico-est nit of all time, but I usually like to keep these keepers sorted in alphabetical order which makes it easier to line up the function signature whenever this list changes!